File: 0645-set-mismatch/0645-set-mismatch.py
class Solution:
    def findErrorNums(self, nums: List[int]) -> List[int]:
        count=[0] *(len(nums)+1)
        for i in range(len(nums)):
            count[nums[i]]+=1
        duplicate=0
        missing=0
        for i in range(1,len(nums)+1):
            if count[i]==2:
                duplicate=i
            if count[i]==0:
                missing=i
        return [duplicate,missing]

File: 0645-set-mismatch/0645-set-mismatch.cpp
class Solution {
public:
    vector<int> findErrorNums(vector<int>& nums) {
        int missing=-1;
        int duplicate=-1;
        int n=nums.size();

        for(int i=1;i<=n;i++){
            int count=0;

            for(int j=0;j<n;j++){
                if(nums[j]==i){
                    count++;
                }
            }

            if(count==0)
                missing=i;

            if(count==2)
                duplicate=i;
        }

        return {duplicate,missing};
    }
};